.github/workflows/pd-test-build-deploy.yaml

```
@@ -133,21 +143,10 @@ jobs:
         with:
           name: pd-artifact
           path: ./dist
-      - name: 'configure ot3 s3 deploy creds and deploy'
+      - name: 'deploy protocol designer to sandbox'
         shell: bash
-        env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.PD_S3_SANDBOX_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.PD_S3_SANDBOX_SECRET }}
-          AWS_DEFAULT_REGION: us-east-1
         run: |
-          aws configure set aws_access_key_id ${{ secrets.PD_S3_SANDBOX_KEY_ID }} --profile identity
-          aws configure set aws_secret_access_key ${{ secrets.PD_S3_SANDBOX_SECRET }} --profile identity
-          aws configure set region us-east-2 --profile identity
-          aws configure set output json --profile identity
-          aws configure set region us-east-2 --profile deploy
-          aws configure set role_arn ${{ secrets.OT_PD_DEPLOY_ROLE }} --profile deploy
-          aws configure set source_profile identity --profile deploy
-          aws s3 sync ./dist s3://sandbox.designer.opentrons.com/${{ env.OT_BRANCH }} --acl=public-read --profile=deploy
+          aws s3 sync ./dist s3://sandbox.designer.opentrons.com/${{ env.OT_BRANCH }} --acl=public-read
```

Does this work?

```suggestion
          aws s3 sync ./dist "s3://sandbox.designer.opentrons.com/${OT_BRANCH}" --acl=public-read
```

* Using shell substitution like `${OT_BRANCH}` instead of GitHub Yaml substitution like `${{ env.OT_BRANCH }}` avoids [this kind of template injection vulnerability](https://docs.zizmor.sh/audits/#template-injection).
* Quotes around the whole shell arg prevent shell injection.

It's a little unclear to me where `OT_BRANCH` is coming from, but I *think* if it's available as `${{ env.OT_BRANCH }}` then it'll always also be available as `${OT_BRANCH}`.

---

Ah yep, 👍🏻 
Thanks!